avakas/avakas.py

```diff
@@ -76,33 +92,128 @@ def read(cls):
     def write(cls):
         """Write version data to a project"""
 
-    def bump(self, bump=None):
+    def bump(self,
+             bump=None,
+             prerelease=False,
+             prerelease_prefix=None,
+             build_date=None):
         """Bump version"""
+
+        original = self._version
+
         if not bump:
             return False
 
         if bump == 'patch':
-            self._version = self._version.next_patch()
+            bump_method = self._version.next_patch
```

Comment:
I may just not be seeing something in another part of the PR but what is the context of these change (function call to property)

Reply:
Oh I'm pretty sure I was handling this differently for a minute inside this function, then went back to how it is. I'll switch this back since there's no reason for the change as it currently is

Reply:
Removed